### PR TITLE
[Bug 20112] unlock cursor on tab-control-alt key

### PR DESCRIPTION
**Pull request by BerndN**

### CODE REVIEW

File: notes/bugfix-120112.md [hunk 1]
```diff
@@ -0,0 +1 @@
+# Unlock cursor on tab-command-alt key
```
Comment (livecodepanos) [inline notes/bugfix-120112.md]: Thank you @BerndN

You have a typo at the name of the bugfix file, it should be `bugfix-20112.md` :)

**Comment by BerndN** [comment]
@livecodepanos Sorry about that, hopefully fixed now

**Comment by livecodepanos** [comment]
Thanks @BerndN 

@livecode-vulcan review ok 65e12e8

**Comment by livecode-vulcan** [comment]
💙 review by @livecodepanos ok 65e12e80af2e3afbf25f8de9f6a8fc0129a2a0e2

**Pull request merged**